collections/tenant/Get Tenant Verified Attribute Revokers.bru

```diff
@@ -24,27 +24,3 @@ headers {
   Authorization: {{JWT-M2M}}
   X-Correlation-Id: {{correlation-id}}
 }
-
-tests {
```

Comment:
@crasherbit I have deleted Bruno tests from this PR and also from the files added in the previous related PR (#2175). In this project, we never used Bruno testing or documentation. I would avoid adding it for new API calls unless we define it as a new testing "standard" approach for the project. Otherwise, it's more code to maintain, and routers' automated tests already test this part with supertest